### ISSUE TITLE
Probleme bei Installation von GitLab Shell

### ISSUE THREAD

**Issue by xarem** [whatwedo/docker-base-images#15]
```
Installing GitLab Shell
mkdir -p /home/git/.ssh: OK
chmod 700 /home/git/.ssh: OK
mkdir -p /data/repositories: OK
chmod ug+rwX,o-rwx /data/repositories: OK
command failed: {"GOPATH"=>"/home/git/gitlab-shell/go_build", "GO15VENDOREXPERIMENT"=>"1"} go install gitlab.com/gitlab-org/gitlab-shell/go/cmd/...
rake aborted!
command failed: bin/compile
/home/git/gitlab/lib/tasks/gitlab/shell.rake:49:in `block (5 levels) in <top (required)>'
/home/git/gitlab/lib/tasks/gitlab/shell.rake:47:in `each'
/home/git/gitlab/lib/tasks/gitlab/shell.rake:47:in `block (4 levels) in <top (required)>'
/home/git/gitlab/lib/tasks/gitlab/shell.rake:18:in `chdir'
/home/git/gitlab/lib/tasks/gitlab/shell.rake:18:in `block (3 levels) in <top (required)>'
/usr/local/bin/bundle:22:in `load'
/usr/local/bin/bundle:22:in `<main>'
Tasks: TOP => gitlab:shell:install
(See full trace by running task with --trace)
```

**Comment by xarem** [comment]
Möglicher Fix in 0570ad088ce014aafd9524d7dd969b738a8a3077

**Comment by n3r0-ch** [comment]
Closing due to deleted GitLab image
